src/pocketmine/Player.php

```diff
@@ -3025,14 +3025,16 @@ public function sendMessage($message){
 			$message = $message->getText();
 		}
 
-		//TODO: Remove this workaround (broken client MCPE 1.0.0)
-		$this->messageQueue[] = $this->server->getLanguage()->translateString($message);
-		/*
-		$pk = new TextPacket();
-		$pk->type = TextPacket::TYPE_RAW;
-		$pk->message = $this->server->getLanguage()->translateString($message);
-		$this->dataPacket($pk);
-		*/
+		if(($translated = $this->server->getLanguage()->translateString($message)) !== ""){
+			//TODO: Remove this workaround (broken client MCPE 1.0.0)
+			$this->messageQueue[] = $this->server->getLanguage()->translateString($translated);
```

Comment:
This is what happens when I copy and paste code I've already written – had to reset the branch because git was being git. Will fix.